alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+                type: string
+                example: TagRuleEngine
+              channel:
+                description: An identifier for the tag or resource associated with the alarm
+                type: string
+                example: CRIO1.System.Health.DiskSpaceMeanUsePercentage
+              resourceType:
+                description: The type of resource associated with the alarm
+                type: string
+                example: Tag
+              displayName:
+                description: Optional display name for a given alarm instance. Display names do not need to be unique.
+                type: string
+                example: Low disk space on tester-1.
+              description:
+                description: Optional description text for a given alarm instance
+                type: string
+                example: Disk space on tester-1 is high. Disk usage was 95% when this alarm was created.
+              keywords:
+                description: Keywords associated with an alarm instance. Alarm instances can be tagged with keywords to make it easier to find them with queries.
+                type: array
+                items:
+                  type: string
+                example:
+                  - keyword1
+                  - keyword2
+              properties:
+                description: Key-value-pair metadata associated with an alarm instance. Alarm instances can be tagged with properties so that they can be more easily found with queries. Additionally, alarm instance properties are used to expand message templates in the notification strategies associated with a given alarm instance.
+                type: object
+                example:
+                  key1: value1
+          required: true
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            title: ID of the created or modified instance
+            properties:
+              instanceId:
+                description: The ID of the created or modified alarm instance.
+                type: string
+                example: 5c40ec55e0d6441168b4c543
+        401:
+          description: Unauthorized
+          schema:
+            $ref: '#/definitions/Error'
+        409:
+          description: For requests which would result in an update to an existing alarm instance, this is returned when the request does not represent a valid transition for the existing alarm instance. This can occur when attempting to clear an alarm instance which is already clear, or when attempting to set an alarm instance which is already set at the given severity level.
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          $ref: '#/responses/Error'
+  /v1/instances/{id}:
+    parameters:
+      - in: path
+        name: id
+        description: Instance ID
+        type: string
+        required: true
+    get:
+      tags: [alarmInstances]
```

Comment:
This route is missing an operationId, so codegen uses the automatic one `V1InstancesIdGet`.